honeybee_radiance/sensor.py

```diff
@@ -31,14 +31,13 @@ def from_dict(cls, sensor_dict):
         .. code-block:: python
 
             {
-            'x': float, 'y': float, 'z': float,
-            'dx': float, 'dx': float, 'dz': float
+            'position': [0, 0, 0],  # array of 3 numbers for the sensor position
+            'direction': [0, 0, 1]  # array of 3 numbers for the sensor direction
```

Comment:
Since there will be 1000s of sensors in most of studies I think keeping the keys shorter might be a good idea. We can use `pos` and `dir` if you think `x` and `dx` is very abstract.

Reply:
Ok. As long as `pos` and `dir` are also the names of the properties on the class, that is fine by me.

Reply:
I guess the one other thing to be aware of is that you are overwriting the native Python dir method when you make it a property on the class. But I guess we do this in other areas like for `face.type`.